docs/ossm/ambient/README.md

```diff
@@ -0,0 +1,340 @@
+3.  Click the Service Mesh Operator.
+4.  Click **IstioCNI**.
+5.  Click **Create IstioCNI**.
+6.  Ensure that the name is `default`.
+7.  Select the `istio-cni` project from the **Namespace** drop-down menu.
+8.  Click on the **Helm Values** -> **cni** -> **ambient** drop-down menu.
+9.  Locate **enabled** parameter and mark the checkbox.
+10. Click **Create**. This action deploys the Istio control plane.  
+    When `State: Healthy` appears in the **Status** column, Istio is successfully deployed.
+
+#### 3.3.3 Creating the Istio CNI resource using the CLI
+
+TBD
+
+### 3.4 Install Ztunnel proxy
+
+#### 3.4.1 Creating the Ztunnel project using the web console
+
+**Prerequisites**
+
+1.  The Red Hat OpenShift Service Mesh Operator must be installed.
+2.  You are logged in to the OpenShift Container Platform web console as cluster-admin.
+
+**Procedure**
+
+1.  In the OpenShift Container Platform web console, click **Home** > **Projects**.
+2.  Click **Create Project**.
+3.  At the prompt, enter a name for the project in the **Name** field. For example, `ztunnel`. The other fields provide supplementary information to the `Istio` resource definition and are optional.  
+    **Note** - The namespace name for `ztunnel` project have to equal the `trustedZtunnelNamespace` parameter in **Istio** configuration.
+4.  Click **Create**. The Service Mesh Operator deploys Istio to the project you specified.
+
+#### 3.4.2 Creating the Ztunnel resource using the web console
+
+**Prerequisites**
+
+1.  The Red Hat OpenShift Service Mesh Operator must be installed.
+2.  You are logged in to the OpenShift Container Platform web console as cluster-admin.
+
+**Procedure**
+
+1.  In the OpenShift Container Platform web console, click **Operators** > **Installed Operators**.
+2.  Select `ztunnel` in the **Project** drop-down menu.
+3.  Click the Service Mesh Operator.
+4.  Click **ZTunnel**.
+5.  Click **Create ZTunnel**.
+6.  Ensure that the name is `default`.
+7.  Select the `ztunnel` project from the **Namespace** drop-down menu.
+10. Click **Create**. This action deploys the Istio control plane.  
+    When `State: Healthy` appears in the **Status** column, Istio is successfully deployed.
+
+#### 3.4.3 Creating the Istio CNI resource using the CLI
+
+TBD
+
+### 3.5 Scoping the Service Mesh with discovery selectors
+
+Service Mesh in Istio ambient mode includes workloads that meet the following criteria:
+
+* The control plane has discovered the workload.
+* The workload has been [labelled appropriately](https://istio.io/latest/docs/ambient/usage/add-workloads/), such that traffic redirection with ZTunnel has been configured.
+
+By default, the control plane discovers workloads in all namespaces across the cluster, with the following results:
+
+* Each proxy instance receives configuration for all namespaces, including workloads not enrolled in the mesh.
+* Any workload with the appropriate pod or namespace injection label receives a proxy sidecar. 
```

Comment:
Since the "sidecar" is not used with Ambient, may be we can rephrase this as follows.

* All the workloads with the appropriate labels would become part of the mesh.

Reply:
Updated.